### PR TITLE
Add IPython 3.2.3 using intel-2015b and Spark 1.6.

### PR DESCRIPTION
**Pull request by ehiggs**

### CODE REVIEW

File: etc/hod/IPython-notebook-3.2.3/start-notebook.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+c.NotebookApp.open_browser = False
+# It is a good idea to put it on a known, fixed port
+c.NotebookApp.port = 8888
+# use IPython.lib.passwd('password') to obtain the password hash
+#c.NotebookApp.password = u'sha1:bcd259ccf...[your hashed password here]'
+
+EOF
+
+# Add the PySpark classes to the Python path (partially taken from pyspark script):
+export PYTHONPATH="$EBROOTSPARK/python/:$EBROOTSPARK/python/lib/py4j-0.8.2.1-src.zip:$PYTHONPATH"
```
Comment (boegel) [inline etc/hod/IPython-notebook-3.2.3/start-notebook.sh]: py4j version has changed:

```
-bash-4.1$ ls  $EBROOTSPARK/python/lib/
py4j-0.9-src.zip  PY4J_LICENSE.txt  pyspark.zip
```

**Comment by boegel** [comment]
@ehiggs: small remark to fix; is this tested?

**Comment by ehiggs** [comment]
This has been tested. :rocket:

**Comment by boegel** [comment]
:shipit:

**Pull request merged**